### DISCUSSION TITLE
Just completed a new fetch with v12 and now the paths for bestiary images look borked?

### DISCUSSION THREAD

**Discussion by Bugmaniac71** [ebullient/ttrpg-convert-cli#23] [Q&A] [answered]
```ad-statblock
title: Goblin Commoner
 dev.ebullient.json5e.qute.ImageRef@1bcbf80a*
*Small humanoid(goblinoid), Any alignment*


Just completed a new fetch with v12 and now the paths for bestiary images looks borked? (now has " dev.ebullient.json5e.qute.ImageRef@1bcbf80a* instead of the image path?

C

**Comment by ebullient** [comment, marked as answer]
oh! I should make another note!

If you're using a custom template, the token structure has changed. That's my fault, I should have done this compatibly.

I've added notes [here](https://github.com/ebullient/json5e-convert-cli#changes-that-impact-generated-templates-and-files), does that help? Should I add more?

**Comment by Bugmaniac71** [comment]
Perfect thanks!
C
